### PR TITLE
SOCK configs

### PR DESCRIPTION
**Pull request by StonesmileGit**
Adds support for SOCK Space Shuttle Orbiter.

**Things I want to do:**
~~Is there a better way to offset the power consumption to be 14 kW?~~
~~Should something be done to the radiator?~~
larger battery? The current battery is the real life size, but the issues with fuel cells turning off makes the time window to fix it very small.

### CODE REVIEW

**Comment by StonesmileGit** [comment]
Can be merged with https://github.com/KSP-RO/RealismOverhaul/pull/2432

**Comment by StonesmileGit** [comment]
https://github.com/KSP-RO/RealismOverhaul/pull/2432 has now been merged, so this can be merged too.

File: GameData/KerbalismConfig/System/Parts.cfg [hunk 1]
```diff
@@ -976,41 +977,72 @@
...
+
+//Add fuel cells to Mk3 here instead
+@PART[mk3Cockpit_Shuttle]:NEEDS[ProfileRealismOverhaul]:AFTER[RealismOverhaul]
+{
+  MODULE
+  {
+    name = ProcessController
+    resource = _FuelCell
+    title = Fuel Cell
+    capacity = 7
```
Comment (Standecco) [inline GameData/KerbalismConfig/System/Parts.cfg:1028]: Shouldn't all shuttles get 21 kW, if you found a source stating 21 kW?

Comment (StonesmileGit) [reply GameData/KerbalismConfig/System/Parts.cfg:1028]: There are 3 fuel cells, total 21 kW

Comment (Standecco) [reply GameData/KerbalismConfig/System/Parts.cfg:1028]: but isn't this adding just one?

**Pull request merged**